lib/hanami/action/mime.rb

```diff
@@ -207,7 +211,14 @@ def format
       #     end
       #   end
       def content_type
```

Comment:
Looking at the changes you made. It looks like it should be covering the RFC now.